src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -34,12 +35,16 @@ function AxisSystem(props: Props): JSX.Element {
   const abscissaScale = getAxisScale(abscissaInfo, width);
   const ordinateScale = getAxisScale(ordinateInfo, height);
 
-  // axisDomains are the complete domains. visibleDomains change with the camera
+  // `axisDomains` are the complete domains; `visibleDomains` change with the camera
   const [visibleDomains, setVisibleDomains] = useState<[Domain, Domain]>([
     abscissaInfo.domain,
     ordinateInfo.domain,
   ]);
 
+  useUpdateEffect(() => {
+    setVisibleDomains([abscissaInfo.domain, ordinateInfo.domain]);
+  }, [abscissaInfo.domain, ordinateInfo.domain]);
+
```

Comment:
This block allows changing domains dynamically in Storybook's "Controls" panel.

Reply:
After some manual testing, it did not seem to affect the app in any way. `useUpdateEffect` means the effect does not run on mount, which would be unnecessary, since on mount, the visible domain's state is already initialised with the axis domains.

Reply:
Why not use a regular `useEffect` ?